tags/faq/lwjgl.ytag

```diff
@@ -2,6 +2,7 @@ type: text
 
 ---
 
-If you're getting `java.lang.UnsatisfiedLinkError: Failed to locate library: lwjgl.dll` when running your project:
-  1. delete `<project dir>/.gradle/loom-cache/launch.cfg`
-  2. regenerate your run configs
+If you're getting `java.lang.UnsatisfiedLinkError: Failed to locate library: lwjgl.dll/dylib/so` when running your project:
+  1. Delete `<project dir>/.gradle/loom-cache/launch.cfg`
+  2. Delete your run configs
+  3. Regenerate your run configs by reimporting the Gradle project
```

Comment:
I would say something along the lines of following the instructions you used to setup the workspace to begin with